docs/findings/sentence-surface.md

```
@@ -336,3 +337,64 @@ Re-verification:
   (`real_tables_session_reports_parity`, `sentence_surface_reports_parity`).
 - union / train / import / predict diffs and bisect+valgrind all green;
   fmt, clippy `-D warnings`, `test --locked --workspace` green.
+## 8. The selection record's row mapping, fixed
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)